### DISCUSSION TITLE
Datetime filter

### DISCUSSION THREAD

**Discussion by jt-atan** [vincjo/datatables#123] [Ideas]
Is it possible to add datetime filtering to handle column with date values?
Filter criteria based on start/min date and end/max date.

**Comment by theoretic** [comment]
A quick workaround: try to place a timestamp into hidden <span> inside <td>.

**Reply by sergeim19** [reply]
@theoretic can you please elaborate? I am using the Date object like in https://vincjo.fr/datatables/docs/client/sort/dates but would like to have the ability to filter for dates in between a certain range.
